index.js

```diff
@@ -406,8 +406,7 @@ TChannelConnection.prototype.onParserError = function onParserError(err) {
 
 TChannelConnection.prototype.nextFrameId = function nextFrameId() {
     var self = this;
-    // TODO this needs to wrap around with 32-bit uint space
-    ++self.lastSentFrameId;
+    self.lastSentFrameId = (self.lastSentFrameId + 1) % 0xffffffff;
```

Comment:
hex !== octal...

this form isn't ambiguous, so if it's linted against, we should fix the linter

also these files are using strict mode already so that's not it